README.md

```
@@ -104,19 +116,18 @@ $options = New-Object Qwiq.Credentials.AuthenticationOptions $uri,Windows
+    FROM WorkItems
     WHERE [System.WorkItemType] = 'Bug' AND State = 'Active'", $false)
 ```
 
 ## Contributing
-**Getting started with Git and GitHub**
 
- * [Setting up Git for Windows and connecting to GitHub](http://help.github.com/win-set-up-git/)
- * [Forking a GitHub repository](http://help.github.com/fork-a-repo/)
- * [The simple guide to GIT guide](http://rogerdudler.github.com/git-guide/)
- * [Open an issue](https://github.com/MicrosoftEdge/Microsoft.Qwiq/issues) if you encounter a bug or have a suggestion for improvements/features
+**Getting started with Git and GitHub**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)